molsysmt/item/molsysmt_Topology/get.py

```diff
@@ -352,7 +352,7 @@ def get_n_entities_from_atom (item, indices='all', check=True):
         _digest_item(item, _form)
         indices = _digest_indices(indices)
 
-    if indices is 'all':
+    if indices == 'all':
```

Comment:
I see the conceptual difference. Thanks for pointing this out.

We should think about how to avoid the future error:
```python
[ins] In [4]: import numpy as np

[ins] In [5]: aa = np.array([0,1,2,3])

[ins] In [6]: aa == 'all'
/home/diego/MyMachine/locals/miniconda/envs/MolSysMT@uibcdf/bin/ipython:1: FutureWarning: elementwise comparison failed; returning scalar instead, but in the future will perform elementwise comparison
  #!/home/diego/MyMachine/locals/miniconda/envs/MolSysMT@uibcdf/bin/python
Out[6]: False

```

Reply:
Thanks 😄 👍🏻